### DISCUSSION TITLE
Anchor link with "→"

### DISCUSSION THREAD

**Discussion by wangzhenux** [twilio-labs/paste#2289] [Q&A] [answered]
Currently I am working on a project where users will click on the link to go to its configuration page in a different Region instead of directly drill down to the current Region. As the page that the link navigates to is in the same domain but different Region, I am wondering if we should use "→" icon as suffix of the anchor to indicate. We have pattern when to use "↗️" suffix when a link navigates users to a different domain. Is there a pattern for users to go to a page under a different Region or different parent page? Thank you!

Visit the [design](https://www.figma.com/file/30SA5yHWj9ckDqVkMecHh3/Messaging-SMS-Regional-2022-Q1?node-id=501%3A25575) for details.

**Comment by nkrantz** [comment, marked as answer]
Hey @wangzhenux! Could you explain where the need for the arrow icon arises? Is it a result of testing you did that showed users wanted something specifying a region link as opposed to other links? That information will help inform us so that we can give you the best guidance. Thanks!

**Reply by loreina** [reply]
@wangzhenux Could you also share an exact link to the Figma screens you're referring to or include screenshots? That would be helpful in understanding the context. Thanks!

**Reply by wangzhenux** [reply]
Sure @loreina. I thought I linked to a specific node but it looks like it broke somehow... I relinked the post or you can use this [link](https://www.figma.com/file/30SA5yHWj9ckDqVkMecHh3/Messaging-SMS-Regional-2022-Q1?node-id=501%3A25575)

**Reply by wangzhenux** [reply]
@nkrantz It's not tested specifically but we should. The problem rose when we went through the phone number configuration workflow (routing specifically), we saw potential confusions when people wanted to go to an active configuration of that phone number under that Region but it directed users to another Region.

A general question I guess: do we have any guideline on using "→" in the anchor?

**Reply by nkrantz** [reply]
Thanks for the context and the Figma link! We don't currently have any guidance for using that arrow icon in anchors. To address your concern of users being directed to a new region unintentionally, you could:
- Clarify the language. Show “Routed to Ireland (IE1)” in plain text, with an Anchor beside it that says “View configuration in IE1".
- Show a [Toast](https://paste.twilio.design/components/toast#neutral-toast) when they've switched regions letting them know what has happened
- (My personal fav) keep the current anchor as is, drop the →, and instead show a tooltip on hover that says “view configuration in IE1”.

Let me know which option you go with!

**Reply by wangzhenux** [reply]
Thank you for your suggestions and information, @nkrantz! It's helpful! I'll let you know once we make a decision!
